### DISCUSSION TITLE
⚠️🧾 DISCLAIMER! 🧾 ⚠️

### DISCUSSION THREAD

**Discussion by deaafrizal** [deaafrizal/forum#6] [Announcements]
**DISCLAIMER:** _please, disini gak harus selalu "bang dea, tanya"._
bisa dibuat secara global aja ya kalau bertanya, forum ini judulnya diskusi, sharing dan tidak selalu terkait atas nama gue.

gue pengen rata bro semua, **no jabatan, no gengsi, no siapa anda siapa saya**. enjoy 🔥

### jika bertanya biasakan hal berikut
- kasih keterengan lagi ngerjain apa, stack apa, versi berapa dan lain-lain. lebih detail lebih bagus.
- kasih tau step yang dilakukan ketika muncul error
- kasih screenshoot code yang dirasa error
- jika tau tau code mana yang error, kasih screenshoot errornya.
- jika sudah solve, dengan hormat tolong beri kabar kembali biar orang bisa pada tau "how to solve" nya, atau minimal apresiasi orang2 yang udah coba bantu jawab pertanyaan lu pada.

**Comment by uratmangun** [comment]
@wdkana bang punya discord atau telegram group nggak?

**Comment by ghifar9** [comment]
ya bang punya discord apa nggakatau grup apa gitu

**Comment by Faizah246** [comment]
ini gw baru masuk banget ke dunia coding ya, bahasa pemrograman juga masih aneh banyak yang ga gw tau, nahh kiranya dari temen temen ada saran harus ngapain gitu kek, gw udah nnton semester 1 cuy univeres, itu yang di fahami cuman 30% cuy, dunia coding emang sesulit ini kah?:(
ini curhat aja sii, siapa tau tiba tiba bisa semangat lagi belajar nyaa

**Reply by fridfn** [reply]
saran nih ya klo baru masuk dunia pemrograman fokusin dulu buat belajar basic dan fundamental dari hal yang pengen di fokusin. contoh kaya pengen bikin web, pelajarin kaya dasar dasar nya dulu dari html css klo udh paham baru ke javascript dan seterusnya.

intinya konsisten terus buat belajar, gpp baru paham dikit. karna kata bang dea "ngoding aja dulu, jago nya belakangan" nanti juga lama lama bakal ngerti klo belajarnya terus konsisten 🔥
